src/io/flutter/run/SdkFields.java

```diff
@@ -216,7 +215,7 @@ public GeneralCommandLine createFlutterSdkRunCommand(
       }
       catch (Exception e) {
         LOG.info(e);
-        FlutterInitializer.getAnalytics().sendException(StringUtil.getThrowableText(e), false);
+        FlutterInitializer.getAnalytics().sendCaughtException("devtools: " + e);
```

Comment:
I assume we can either switch this to LOG.error() - and not send the exception separately - or keep it at LOG.info() and not send the exception.

It's a fairly big hammer to just catch `Exception`. Are there a few types that a thrown? If so, we should specialize what we do here based on the type of exception.

Reply:
It looks like there are at least two - `ExecutionException` and `TimeoutException` - but I also made this broad since missing an exception type would result in blocking running an app, and we don't need different user experiences based on the exception type.

Reply:
OK, as long we you think we should handle the exception classes uniformly.